dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -247,6 +247,28 @@ def _add_helper(self, other1, other2):
         if "num_negatives" in self.__calculations.keys():
             self.num_negatives = other1.num_negatives + other2.num_negatives
 
+    def profile(self):
+        """
+        Property for profile. Returns the profile of the column.
+        :return:
+        """
+
+        profile = dict(
+            min=self.np_type_to_type(self.min),
```

Comment:
since `num_zeros` and `num_negatives` are calculated here in numericstats, I think they should also be. part of the profile here too. Text which doesn't have it can simply pop them.

Reply:
we can discuss about this. IMO, we don't want to update first and pop out later. That's why I don't update these two stats in numeric, but only update them for float and int